pkg/gitrepo/repo_test.go

```diff
@@ -56,14 +89,16 @@ func TestResolveVersion(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	for i, test := range testCases {
+	for i, tc := range testCases {
 		t.Run(strconv.Itoa(i), func(t *testing.T) {
-			version, err := repo.ResolveVersion(ctx, test.inputRef)
+			t.Log(tc.name)
```

Comment:
Mind blown. Didn't know about this. :+1:

Reply:
Yeah turned out to be very helpful with debugging.